pdfalyzer/util/argument_parser.py

```diff
@@ -50,6 +50,10 @@
                     const='bin',
                     help='extract all binary streams in the PDF to separate files (requires pdf-parser.py)')
 
+export.add_argument('-json',
```

Comment:
deleted my previous comment, now see [my new comment](https://github.com/fdattein/pdfalyzer/pull/1#issuecomment-3002183775) about bumping yaralyzer to get access to the existing `--export-json` argument